### PR TITLE
Survey refactor expansion UI changes

### PR DESCRIPTION
**Pull request by sebastianbarry**
I previously had this PR pointed to `places-survey` branch (https://github.com/sebastianbarry/e-mission-phone/pull/7) but after a short discussion with Shankari I am redoing this PR for `survey_refactor_expansion` branch:

# Testing:
Part 1: **Left justify entry labels**

Part 2: **Confirm delete entry popup**
https://user-images.githubusercontent.com/61334340/226062221-34999dea-69f1-422f-af9d-5e48ccb71265.mov

### CODE REVIEW

**Comment by sebastianbarry** [comment]
@shankari does this PR look correct?

**Comment by shankari** [comment]
At least it is to the right branch?!

File: www/css/main.diary.css [hunk 1]
```diff
@@ -494,19 +494,22 @@ enketo-notes-list, .notes-list {
 }
 
 .notes-list {
-  padding: 0 5px 5px 5px;
+  padding: 0 15px 5px 15px;
```
Comment (JGreenlee) [inline www/css/main.diary.css:497]: To me, this puts the trash can a little too far in. How about 15 on the left and 10 on the right?
```suggestion
  padding: 0 10px 5px 15px;
```
Here's what that looks like
image

**Comment by JGreenlee** [comment]
Real quick, I just patched up the CSS so that the vertical alignment was retained

Left the pop-up as-is since it looks great. We will want to internationalize the text though.

File: www/templates/diary/place_list_item.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
         <!-- Start Time Tag -->
         <!-- <div class="start-time-tag">{{trip.display_end_time}}</div> -->
         <div class="diary-card short" style="padding-top: 10px;">
-            <div style="width: 100%; margin: auto; padding: 5px">
```
Comment (JGreenlee) [inline www/templates/diary/place_list_item.html:7]: Why was this padding removed?
We can do without it - but if we do remove it, then we have to update the card height values in `infinite_scroll_list.js > getTripHeight()`

**Comment by shankari** [comment]
I am going to revert the padding change and merge tonight.

@sebastianbarry you need to create a follow-on change that:
- discusses/resolves the padding
- addresses the i18n of the new strings

File: www/templates/diary/place_list_item.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
...
+            <div style="width: 100%;">
```
Comment (shankari) [inline www/templates/diary/place_list_item.html:7]:
```suggestion
            <div style="width: 100%; margin: auto; padding: 5px">
```

File: www/templates/diary/place_list_item.html (hunk 2 above)
Comment (shankari) [reply www/templates/diary/place_list_item.html:7]: Reverting. @sebastianbarry please address this comment on Monday

**Review by shankari — APPROVED** [review]

**Pull request merged**